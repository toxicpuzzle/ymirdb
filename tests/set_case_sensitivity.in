SET a 100 99 88
SET A 100 9 9
SET aB 100 9
SET Ab 12 23 aB
LIST ENTRIES
GET a
GET A
GET aB
GET Ab
bye