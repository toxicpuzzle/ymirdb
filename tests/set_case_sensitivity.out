> ok

> ok

> ok

> ok

> Ab [12 23 aB]
aB [100 9]
A [100 9 9]
a [100 99 88]

> [100 99 88]

> [100 9 9]

> [100 9]

> [12 23 aB]

> bye
